docs/faq/users/delta.rst

```diff
@@ -0,0 +1,228 @@
+Recipe: Run a Job on Delta Using an Allocation
+----------------------------------------------
+
+This recipe assumes that you have decided to use your allocation
+for Delta to run one of your HTCondor jobs.  It takes you step by
+step through the process of Bringing Your Own Resources (BYOR) in the
+form of an allocation to an OSG Portal access point and using that
+resource to run your HTCondor job.  In what follows, we refer to the
+named set of resources leased from that allocation as an *annex*.
+
+In this recipe, we assume that the job has not yet been placed at an
+OSG Portal access point when we begin.
+
+Ingredients
+===========
+
+- An
+  `OSG Portal account <https://portal.osg-htc.org/application>`_
+  and password.
+- An HTCondor job submit file (:doc:`example.submit <annex-example-job>`).
+- An allocation for Delta.
+- Command-line login access to Delta (see
+  `NCSA's instructions for gaining access <https://wiki.ncsa.illinois.edu/display/USSPPRT/NCSA+Allocation+and+Account+Management>`_).
+  We'll use ``LOGIN_NAME`` to refer to your login name on Delta.
+- A name for your Delta annex (example).  By convention,
+  this is the name of the submit file you want to run, without its extension.
+
+Assumptions
+===========
+
+- You want to run the job described above on Delta.
+- The job described above fits within the capabilities of the
+  `Delta queue <https://docs.ncsa.illinois.edu/systems/delta/en/latest/user_guide/running_jobs.html#partitions-queues>`_
+  you wish to run it on.
+
+Preparation
+===========
+
+First, you will need to determine the project ID of your allocation on
+Delta.  If you already know your project ID, you can skip this
+section.  If not, log in to ``login.delta.ncsa.illinois.edu`` in a terminal and run the
+following command.  (Don't copy the ``$``; in this and other examples
+further down the page, the ``$`` just signifies something you type in,
+rather than something that the computer prints out.)
+
+.. code-block:: text
+
+    $ accounts
+
+Pick one of the entries in the ``Project`` column.
+For the rest of these instructions, we'll use ``PROJECT_ID`` where you
+need to put that entry.
+
+Instructions
+============
+
+1. Log into the OSG Portal Access Point
+'''''''''''''''''''''''''''''''''''''''
+
+Log into an OSG Portal access point (e.g., ``ap20.uc.osg-htc.org`` or
+``ap21.uc.osg-htc.org``) using your OSG Portal account and password.
+
+2. Submit the Job
+'''''''''''''''''
+
+Submit the job on the access point, indicating that you want it to run
+on your own resource (the Delta allocation, in this case) with the
+``--annex-name`` option:
+
+.. code-block:: text
```

Comment:
Not a big deal, but we usually use `.. code-block:: console`  for shell commands.  One day I want to style console code blocks so you can easily select the command without the promplt

Reply:
This is consistent with the rest of the pages, and necessary -- AFAICT -- to make it consistent with the look of the `parsed-literal` block, which is necessary -- AFAICT -- to get the bold text that's actually used on-screen.

Reply:
We'll work on this later. :)